src/arch/xtensa/smp/cpu.c

```diff
@@ -115,5 +115,5 @@ void cpu_power_down_core(void)
 	free_heap(RZONE_SYS);
 
 	while (1)
-		arch_wait_for_interrupt(0);
+		asm volatile("waiti 0");
```

Comment:
probably best to stick a comment here stating why we do this directly to stop anyone doing a reverse change later on.